src/rest_framework_dso/embedding.py

```diff
@@ -224,6 +224,39 @@ def _expand_parse_error(allowed_names, field_name, prefix=""):
         )
 
 
+def get_serializer_lookups(serializer: serializers.BaseSerializer, prefix="") -> List[str]:
```

Comment:
Nice! Lot of thinking behind this I guess.

Reply:
tx :-)